src/EditorFeatures/VisualBasicTest/QualifyMemberAccess/QualifyMemberAccessTests.vb

```diff
@@ -353,6 +410,19 @@ CodeStyleOptions.QualifyMethodAccess)
 CodeStyleOptions.QualifyMethodAccess)
         End Function
 
+        <WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)>
+        Public Async Function QualifyMethodAccess_NotSuggestedOnLocalVarInObjectInitializer() As Task
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)